test/jasmine/tests/snapshot_test.js

```diff
@@ -125,23 +129,54 @@ describe('Test Snapshot.clone', function() {
                 tileClass: 'notarealclass'
             };
 
-            var vanillaPlotTile = Snapshot.clone(dummyGraphObj, vanillaOptions);
+            var vanillaPlotTile = Plotly.Snapshot.clone(dummyGraphObj, vanillaOptions);
             expect(vanillaPlotTile.data[0].x).toEqual(data[0].x);
             expect(vanillaPlotTile.layout).toEqual(layout);
             expect(vanillaPlotTile.layout.height).toEqual(layout.height);
             expect(vanillaPlotTile.layout.width).toEqual(layout.width);
         });
 
         it('should set the background parameter appropriately', function() {
-            var pt = Snapshot.clone(dummyGraphObj, {
+            var pt = Plotly.Snapshot.clone(dummyGraphObj, {
                 setBackground: 'transparent'
             });
             expect(pt.config.setBackground).not.toBeDefined();
 
-            pt = Snapshot.clone(dummyGraphObj, {
+            pt = Plotly.Snapshot.clone(dummyGraphObj, {
                 setBackground: 'blue'
             });
             expect(pt.config.setBackground).toEqual('blue');
         });
     });
+
+    describe('toSVG', function() {
+        var gd;
+
+        beforeEach(function() {
+            gd = createGraphDiv();
+        });
+
+        afterEach(destroyGraphDiv);
+
+
+        it('should not return any nested svg tags of plots', function(done) {
+            Plotly.plot(gd, subplotMock.data, subplotMock.layout).then(function() {
+                return Plotly.Snapshot.toSVG(gd);
+            }).then(function(svg) {
+                var splitSVG = svg.split('<svg');
+
+                expect(splitSVG.length).toBe(2);
```

Comment:
Could we make this test even better by searching for one and only one `<svg>` and one and only one `</svg>` ?